infrastructure/quick-deploy/aws/parameters.tfvars

```diff
@@ -197,8 +197,8 @@ eks_managed_node_groups = {
       }
     }
     capacity_type = "ON_DEMAND"
-    min_size      = 1
-    desired_size  = 1
+    min_size      = 5
+    desired_size  = 5
```

Comment:
You could make it depend on whether we deploy the sharded mongodb or not?

Reply:
I guess so, I agree on the fact that we should design the "MongoDB nodes" to be kind of proportionnal with the resources needed for MongoDB. We'll talk about this

Reply:
You could keep min_size and desired_size to 1 and rely on the cluster autoscaler to start more nodes when you have more shards and replicas.

Reply:
Seems smart. Is there a similar mechanism for GCP deployment ?